src/Views/Comparator.elm

```diff
@@ -152,11 +152,18 @@ comparatorView config =
                     )
                 |> RE.combine
     in
-    [ [ ( "Sous-scores", Subscores )
-      , ( "Impacts", IndividualImpacts )
-      , ( "Étapes", Steps )
-      , ( "Total", Total )
-      ]
+    [ ((if Session.isLoggedIn config.session then
+            [ ( "Sous-scores", Subscores )
+            , ( "Impacts", IndividualImpacts )
+            ]
+
+        else
+            []
+       )
+        ++ [ ( "Étapes", Steps )
+           , ( "Total", Total )
+           ]
```

Comment:
By default on the Food page with an empty product, we have an empty single "Étapes" tab with nothing at all under it:

![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/8a780817-af20-4932-b504-c4fdcef2f839)

We shouldn't render anything if we don't have anything to render yet.

Reply:
Indeed. As it's the case also on the current v2, I'll create a notion card to address that at a later stage, thanks for the heads up.